lib/check-link.js

```diff
@@ -4,18 +4,27 @@ import https from 'https'
 import { utils } from './utils.js'
 
 export const agent = {
-  http: new http.Agent({ rejectUnauthorized: false }),
+  http: new http.Agent(),
```

Comment:
`http.Agent` doesn't support the `rejectUnauthorized` option, which only applies to `https`

Reply:
great catch :)